tools/build.py

```diff
@@ -194,6 +196,7 @@ def build_options_append(cmakeopt, cliarg):
     build_options_append('FEATURE_MEM_STRESS_TEST', arguments.mem_stress_test)
     build_options_append('FEATURE_PROFILE', arguments.profile)
     build_options_append('FEATURE_REGEXP_STRICT_MODE', arguments.regexp_strict_mode)
+    build_options_append('REGEXP_RECURSION_LIMIT_N', arguments.regexp_recursion_limit)
```

Comment:
What is that `_N`? I cannot really see the need for it.

Reply:
`_N` is indicating that this is a number. You are right, there is no real need for that, I will change it and update the PR.